interfaces/python_sdist/setup.py

```diff
@@ -92,28 +93,95 @@ def cythonize(extensions):
 elif BOOST_INCLUDE is not None:
     include_dirs.append(BOOST_INCLUDE)
 
-if sys.platform != "win32":
-    extra_compile_flags = ["-std=c++11"]
-    sundials_configh = {
-        "SUNDIALS_USE_GENERIC_MATH": "#define SUNDIALS_USE_GENERIC_MATH 1",
-        "SUNDIALS_BLAS_LAPACK": "/* #undef SUNDIALS_BLAS_LAPACK */"
-    }
-    sundials_cflags = ["-w"]
-    sundials_macros = []
+
+def configure_build():
+    boost_version = ""
+    boost_locs = (os.environ.get("BOOST_INCLUDE", None), BOOST_INCLUDE, "/usr/include",
```

Comment:
Is all of what follows just to figure out whether we have Boost >= 1.56? That's pretty old (2014), and I'd say that just dropping it is definitely an option. Though I we might want to optionally use some newer boost feature in the future, in which case having this available would be useful.

Reply:
The Linux wheels are built (by default) in the `manylinux2014` container which is built on CentOS 7, which has maybe 1.54 in the repos (older than 1.56 anyways)? Santosh posted an issue about missing demangle not that long ago as well. Anyways, as you say, this isn't much code and it will eventually allow us to use newer Boost features.